controllers/controller.js

```diff
@@ -16,4 +19,19 @@ function getEndpoints(req, res, next) {
   res.status(200).send({ endpoint });
 }
 
-module.exports = {getTopics, getEndpoints}
\ No newline at end of file
+function getArticleById(req, res, next) {
+    const articleArr = req.params
```

Comment:
`articleArr` doesn't describe `req.params` very clearly as a variable name,  as it's not an array
It would make sense to take the relevant id off the `req.params` object here